brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-3168 Resolve reusable block issue ( Also related Astra custom layout issues ) 1 to 22 blocks.

### Description

Resolved reusable block issue in : - 
### P1 - blocks ###
1. Counter
2. FAQ
3. FAQ Child
4. Forms
5. Forms Accept
6. Forms Checkbox
7. Forms Date
8. Forms Email
9. Forms Hidden
10. Forms Name
11. Forms Phone
12. Forms Radio
13. Forms Select
14. Forms Textarea
15. Forms Toggle
16. Forms Upload
17. Forms URL
### P2 - blocks ###
1. Blockquote
2. Column
3. Columns
4. CF7 Styler

### Screenshots

- NA

### Types of changes




- Optimisation - Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

### Test case 1 ###
- Drag and drop blocks mention in above list on page.
- Make it reusable block.
- Click on + icon, Go to Reusable section and select the same block that you make it reusable
- After drag and drop same reusable on same page, page as well as block should not break.
- Now add some styling to reusable block, update page and make sure styling are applying properly in editor as well as on frontend.
### Test case 2 ###
- Install Astra Addon 4.1.6
- Go to Astra custom layout.
- Select Custom option.
- Drag and drop above mention block and make sure it does not break in editor.

### Checklist:
